lib/parity/environment.rb

```diff
@@ -132,7 +132,7 @@ def run_migration?
     end
 
     def rails_app?
-      Kernel.system("command -v rake && rake -n db:migrate")
+      Kernel.system('[ -d "db/migrate" ] && [ -f "Rakefile" ]')
```

Comment:
What about `File.directory?("db/migrate") && File.file?("Rakefile")`?

I'm mostly thinking of portability -- what shell will `system()` pick?